src/nerf_loader.cu

```diff
@@ -864,7 +874,10 @@ void NerfDataset::update_metadata(int first, int last) {
 	}
 
 	metadata_gpu.enlarge(last);
-	CUDA_CHECK_THROW(cudaMemcpy(metadata_gpu.data() + first, metadata.data() + first, n * sizeof(TrainingImageMetadata), cudaMemcpyHostToDevice));
+	if (!in_cpu_ram) {
```

Comment:
This seems wrong to me. The metadata is still stored on the GPU -- and it's so small that it wouldn't make use to offload to CPU ram anyway.